libs/gl-client/src/lnurl/pay/mod.rs

```diff
@@ -123,6 +173,47 @@ fn ensure_amount_is_within_range(
     Ok(())
 }
 
+//LUD-16: Paying to static internet identifiers.
+pub fn parse_lightning_address(lightning_address: &str) -> Result<String> {
+    let lightning_address_components: Vec<&str> = lightning_address.split("@").collect();
```

Comment:
Add a check that this results in two parts. Abort if not.

Reply:
Added the check